qupath-gui-fx/src/main/java/qupath/lib/gui/commands/PathObjectGridView.java

```diff
@@ -580,21 +581,20 @@ protected void layoutChildren() {
 			}
 			
 			int padding = 5;
-			
-			int w = (int)getWidth();
-//			int h = (int)getHeight();
 			int dx = imageSize.get() + padding;
+			int w = Math.max(dx, (int)getWidth());
 			int nx = (int)Math.floor(w / dx);
+			nx = Math.max(1, nx);
 			int spaceX = (int)((w - (dx) * nx) / (nx)); // Space to divide equally
 			
-			int x = spaceX/2;
+			int x = spaceX/2; //1 space increment to divide equally
```

Comment:
I'm not sure what this comment means (although tbf I'm not sure what the one above it means, and there's a good chance I wrote that one)

Reply:
Ah missed one, removed now :)